react/src/sagas/OpenidConnectFreja.js

```diff
@@ -13,8 +13,12 @@ export function* checkNINAndShowFrejaModal () {
     if (!nin) {
       let inputNin = document.querySelector('input[name=norEduPersonNIN]');
       if (!inputNin) {
-        // Form is always created, only hidden. Must be running tests.
-        nin = 'testing';
+        const unconfirmed = document.getElementById('eduid-unconfirmed-nin');
+        if (unconfirmed) {
```

Comment:
this one is slightly different than the other two above - intentionally?

Reply:
The logic seems the same, the module was developed by Johan, and I have respected the basic structure...